website/data/open-source/projects.yml

```diff
@@ -31,6 +31,9 @@ UI/UX:
 Data Science & Engineering:
   - dr5hn/countries-states-cities-database
 
+Full Stack Development:
+  - Ayush7614/Bundli-Frontend    
+
```

Comment:
Thanks for your contribution @Ayush7614 💙 

Since Bundli seems more focussed towards learning, can you please move it under Learning & Education?